scout/models/case/case_loading_models.py

```diff
@@ -499,6 +500,20 @@ def set_synopsis(cls, synopsis: Optional[Union[str, List]]) -> Optional[str]:
     @model_validator(mode="before")
     @classmethod
     def format_build(cls, values) -> "CaseLoader":
```

Comment:
Shouldn't this be named `format_rna_build` and the function down (line 516) `format_build`?

Reply:
Also these 2 functions have some common code. Couldn't they be unified somehow?

Reply:
Sure, one way would be to break out the genome string check to a separate helper..